app/controllers/accountants_controller.rb

```diff
@@ -1,10 +1,14 @@
 # Controller pertaining to accountant functions, usually fulfillments.
 class AccountantsController < ApplicationController
-  def index; end
+  before_action :confirm_admin_user
+  
+  def index
+    @patients = Patient.where('fulfillment.fulfilled' => true)
+    @patients = Kaminari.paginate_array(@patients).page(params[:page]).per(25)
+  end
 
   def search
-    @results = Patient.search params[:search]
-
+    @results = Patient.where('fulfillment.fulfilled' => true).where(:name => /#{params[:search]}/i)
```

Comment:
I'd suggest taking advantage of the `Patient.search` method here and chaining it against the `where`. e.g. `Patient.search(params[:search]).where('fulfillment.fulfilled' => true)` if memory serves. (s/o to @KatSDC for writing a slamming search algo!)

Reply:
Can you tell me where that method definition is handled, i'd like to see if it supports cases.

Reply:
The search internals for patients are here -- https://github.com/DCAFEngineering/dcaf_case_management/blob/master/app/models/concerns/patient_searchable.rb